pkg/config/config.go

```diff
@@ -328,6 +328,11 @@ type EngineConfig struct {
 	// this slice takes precedence.
 	HooksDir attributedstring.Slice `toml:"hooks_dir,omitempty"`
 
+	// Location of CDI configuration files. These define mounts devices and
+	// other configs according to the CDI spec. In particular this is used
+	// for GPU passthrough.
+	CdiSpecDir attributedstring.Slice `toml:"cdi_spec_dir,omitempty"`
```

Comment:
FWIW we have this option already in CRI-O: https://github.com/cri-o/cri-o/blob/2a20780b7b91fecb27c2663558bb06cc7f817d14/internal/criocli/criocli.go#L963-L968

Reply:
@saschagrunert are you saying this can be accomplished with CRI-O config? Or are you saying this should follow the same pattern fo CRIO? Or are you saying this is a useful thing to have?

Reply:
should probably rename to cdi-spec-dirs to match